inc/dashboard/provider.class.php

```diff
@@ -338,41 +338,47 @@ static function nbTicketsGeneric(
             $params['icon']  = Ticket::getIcon();
             $params['label'] = __("Incoming tickets");
             $skip = true;
+            //no break
```

Comment:
Do not know why

Reply:
avoid repetition mainly, maybe a bad design choice on my side.
You can rewrite it, i'm not against

Reply:
no, ok for me, I better understand now